task2.rb

```diff
@@ -0,0 +1,31 @@
+# Task #2
+# Написати програму, яка буде пропонувати користувачу ввести певне число.
+# У відповідь на введене число, програма має виводити в консоль задане число помножене на 256 та пропонувати ввести ще одне число.
+# Наведена вище логіка має повторюватись безліч кількість разів.
+# У випадку, якщо програма отримала число, яке містить у своєму записі більше ніж одну одиницю (1),
+# потрібно вивести користувачу якесь довільне повідомлення про завершення виконання цієї програми та зупинити програму.
+#
+# Приклад взаємодії користувача з цією програмою:
+# Введіть число:
+# 1
+# Результат виконання: 256
+# Введіть число:
+# 171
+# Умови недотримані, зупиняю виконання програми.
+puts 'Please, enter any number'
+
+number = gets.chomp
+
+def check_numbers(number)
+  acc = 0
+  arr_numbers = number.to_s.split(//)
+  arr_numbers.each { |i| acc += 1 if i.to_i == 1 }
+  acc > 1 ? false : true
+end
+
+while check_numbers(number)
+  puts "Result #{number.to_i * 256}, please enter new number"
+  number = gets.chomp
+end
+
+puts 'Oh no, your used \'1\' a lot, sorry. Lucky next time!'
```

Comment:
```suggestion
puts "Oh no, your used '1' a lot, sorry. Lucky next time!"
```

Для того щоб не використовувати бекслеші для передачі `'`, просто юзай подвійні скобки, бо разом з ними важко читати наповнення стрінги